src/compiler/transformers/classFields.ts

```diff
@@ -1341,6 +1534,15 @@ namespace ts {
         }
     }
 
+    function createPrivateStaticFieldInitializer(variableName: Identifier, initializer: Expression | undefined) {
+        return factory.createAssignment(
+            variableName,
+            factory.createObjectLiteralExpression([
+                factory.createPropertyAssignment("value", initializer || factory.createVoidZero())

```

Comment:
Should we do the `{ __proto__: null }` trick here? Not really sure if it's worth it.

Reply:
I guess it's a "no"? 😄

Reply:
Next on my to do list :)

Reply:
As we discussed we'll skip this at least for now.